modules/oe_content_call_proposals/src/Plugin/Validation/Constraint/CallProposalsJournalTitleLength.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
I don't think we need a custom validator for this, we already have the maxlength module enabled so we can use that instead.

Reply:
I removed the custom validator and set the `maxlength` related constraint.